LDAP-primer.md

```diff
@@ -338,3 +361,26 @@ exist in the entry, so renaming an entry to a RDN that is not already present wo
 operations: one to add a new value and one to perform a renaming. To account for this situation,
 __modify DN__ will add the new naming attribute and value to the renamed entry automatically,
 and will also optionally delete the previous naming attribute-value pair.
+
+## 3.4. Extensions
+
+LDAP can be extended by servers in two main ways, using __controls__ and __extended operations__.
+There are lots of RFCs which standardize certain controls and extended operations.
```

Comment:
Slightly reformulated to emphasize that it's the operations that can be extended, not the data model.